apps/docs/src/app/platform/component-docs/platform-forms/platform-textarea/platform-textarea-examples/platform-textarea-counter-example.component.html

```diff
@@ -0,0 +1,77 @@
+<main class="fd-page">
+    <div *ngIf="true" class="fd-page__content fd-has-background-color-neutral-2">
+        <fdp-form-group #fg2 [multiLayout]="true" [formGroup]="form">
+            <fdp-form-field
+                #ff2
+                [id]="'detailedDescription'"
+                label="Detailed description"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+                [required]="true"
+                [validators]="textareaValidator"
+            >
+                <fdp-textarea
+                    [formControl]="ff2.formControl"
+                    [name]="'detailedDescription'"
+                    [growingMaxLines]="3"
+                    [growing]="true"
```

Comment:
growing should take into account when user clicks enter as a line

Reply:
I don't quite understand how to click enter, is it also there in the UI5 textarea sample?